packages/types/src/channels.ts

```diff
@@ -33,7 +33,7 @@ export type Channel = {
     update_at: number;
     delete_at: number;
     team_id: string;
-    type: ChannelType;
+    type: ChannelType | string;
```

Comment:
if the type you need is not currently present, please add it to the ChannelType interface